### ISSUE TITLE
Field Types

### ISSUE THREAD

**Issue by Christianuss** [RohanNagar/op-connect-sdk-java#6]
```
com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `com.sanctionco.opconnect.model.Type` from String "ADDRESS": not one of the values accepted for Enum class: [STRING, TOTP, MONTH_YEAR, DATE, MENU, EMAIL, CONCEALED, URL]
```

When retrieving an item that has an `ADDRESS` or `PHONE` field. I don't know if these fields can be used when adding new items or if the values can be read (since the [docs](https://support.1password.com/connect-api-reference/#item-field-object) don't mention them), but it would probably be useful to at least handle these field types somehow.

The same exception is thrown when retrieving an item that has a one-time password field. The docs state the field type is `TOTP`, but apparently it is `OTP`.

**Comment by RohanNagar** [comment]
Fixed all of these in 83deee6

**Comment by RohanNagar** [comment]
Thanks for reporting these bugs. I released version `0.2.1` which fixes the 3 issues you opened.
